src/stylesheets/components/_card.scss

```diff
@@ -53,7 +53,8 @@
   @include u-border($theme-card-border-width, $theme-card-border-color);
   @include u-display("flex");
   @include u-height("full");
-  @include u-flex("column");
+  @include u-flex("row");
+  align-items: center;
```

Comment:
This has unintended consequences in the default component

**Current**
![image](https://user-images.githubusercontent.com/3385219/122610694-aedd6d80-d045-11eb-9986-fda6b02c8ea2.png)

**Expected**
![image](https://user-images.githubusercontent.com/3385219/122610738-c0267a00-d045-11eb-98cf-8fceb78faf24.png)
[Preview card on develop](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/components/detail/card--default.html)

Reply:
Sorry about that.  Pushed up a fix for that.